examples/recover/recover.go

```diff
@@ -0,0 +1,49 @@
+// A `recover` means recovering from a `panic`, either from a "business" or "built-in" panic.
+// We want to recover if we want to handle a panic, stopping it from propagating upwards.
+
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+
```

Comment:
There aren't many comments in this sample. Is this intentional?

Reply:
Added some comments in 47eed5ec17c1925520196d718befa82e69f5ccbc.
Coming from Java's "exceptions world", the recover seems a bit trivial, so I'm not sure what more info should I add.